### PR TITLE
{lib}[GCCcore/6.4.0] libpthread-stubs v0.3

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2087.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ea0ec9fb52b1addc74322ddccff48403 for a full test report.

File: easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.3-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ a small footprint, latency hiding, direct access to the protocol, improved
+ threading support, and extensibility.
+"""
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['3031f466cf0b06de6b3ccbf2019d15c4fcf75229b7d226a711bc1885b3a82cde']
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
```
[the remark this answers is not in the document]
Comment (JackPerdue) [reply easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.3-GCCcore-6.4.0.eb]: totally agree... will make repairs after submitting what I have plated at the moment (about 200 packages0

[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/l/libpthread-stubs/libpthread-stubs-0.3-GCCcore-6.4.0.eb]: @JackPerdue Hold on... Are you planning to bombard us with 200 PRs?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2724.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/07f3be2ea9c933b227602c3a3f4accab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2121.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/33c474356a9900f8081b0aaed1d3dc91 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**